backends/tc/ebpfCodeGen.cpp

```diff
@@ -212,6 +212,60 @@ void PNAArchTC::emitInstances(EBPF::CodeBuilder *builder) const {
     builder->newline();
 }
 
+void PNAArchTC::emitGlobalFunctions(EBPF::CodeBuilder *builder) const {
+    const char *code =
+        "u32 getPrimitive32(u8 *a, int size) {\n"
```

Comment:
Small one, since this will be in an include file:

```suggestion
        "static inline u32 getPrimitive32(u8 *a, int size) {\n"
```